Adamant/Modules/Account/AccountViewController/AccountViewController.swift

```diff
@@ -67,6 +67,10 @@ final class AccountViewController: FormViewController {
     
     private var notificationsSet: Set<AnyCancellable> = []
     
+    private var currentWallet: WalletServiceCompose {
```

Comment:
Are you sure with naming? currentWallet looks like entity, not service. just a nit